src/server/standalone.js

```diff
@@ -591,7 +593,14 @@ function StandAloneServer(gmeConfig) {
     logger.debug('initializing static server');
     __app = new Express();
 
-    __database = new Mongo(logger, gmeConfig);
+    if (gmeConfig.storage.database.type === 'mongo') {
+        __database = new MongoAdapter(logger, gmeConfig);
+    } else if (gmeConfig.storage.database.type === 'redis') {
+        __database = new RedisAdapter(logger, gmeConfig);
+    } else if (gmeConfig.storage.database.type === 'memory') {
+        __database = new MemoryAdapter(logger, gmeConfig);
+    }
```

Comment:
I would add a comment: `// no else branch type is checked from the configuration validator see ...`